engine.js

```diff
@@ -17,13 +17,13 @@ var filter = function(array) {
 module.exports = function (options) {
 
   var types = options.types;
-  var format = options.format || '[emoji] [name]: [subject]';
+  var format = options.format || '[name]: [subject] [emoji]';
```

Comment:
I think the order of the emoji should be in the front as it can be easily changed by `format` setting. The emoji is basically the heart of this package and we should keep it as the current behavior.

Reply:
Done

Reply:
Oops, I mean that we should keep the default order. Sorry if I'm not making it clear.

Reply:
Oh, well, that's the whole point of the PR. :) 

Emojis are not part of the standard. And so keeping them in front breaks the tool chain.

Reply:
Yep but I think changing them with one line `format` is a simple enough hack. But we can sure change to lower case by default in the type names 😉